docs/AboutUs.adoc

```diff
@@ -11,46 +11,40 @@ We are a team based in the http://www.comp.nus.edu.sg[School of Computing, Natio
 
 == Project Team
 
-=== John Doe
-image::damithc.jpg[width="150", align="left"]
-{empty}[http://www.comp.nus.edu.sg/~damithch[homepage]] [https://github.com/damithc[github]] [<<johndoe#, portfolio>>]
-
-Role: Project Advisor
-
 '''
 
-=== John Roe
-image::lejolly.jpg[width="150", align="left"]
+=== Thomas Tan
+image::thomastanck.jpg[width="150", align="left"]
 {empty}[http://github.com/lejolly[github]] [<<johndoe#, portfolio>>]
```

Comment:
we should be updating the GitHub URLs too right?

Reply:
Updated